specification/storage/data-plane/Azure.Storage.Files.DataLake/readme.md

```diff
@@ -30,58 +30,13 @@ use-internal-constructors: true
 add-credentials: true
 ```
 
-### Tag: package-2018-06-preview
```

Comment:
Is there a reason we're deleting these settings when the those preview files still exist?

Reply:
Yeah, we'll need to change this to make the Avocado errors go away before merging.  Example:
> The swagger JSON file is not referenced from the readme file.
readme: [storage/data-plane/Azure.Storage.Files.DataLake/readme.md](https://github.com/Azure/azure-rest-api-specs/blob/6d30ae6b02efcc07a888e3db5c181678cf433d4f/specification/storage/data-plane/Azure.Storage.Files.DataLake/readme.md)
json: [Azure.Storage.Files.DataLake/preview/2018-06-17/DataLakeStorage.json](https://github.com/Azure/azure-rest-api-specs/blob/6d30ae6b02efcc07a888e3db5c181678cf433d4f/specification/storage/data-plane/Azure.Storage.Files.DataLake/preview/2018-06-17/DataLakeStorage.json)

Reply:
Fixed.